### PR TITLE
push code

### PR DESCRIPTION
**Pull request by ssola**

### CODE REVIEW

File: tools/data/transfer.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+    task_id = uuid.uuid1()
+
+    print "Task ID: {}".format(str(task_id))
+    print "From: {}".format(args.origin_server)
+    print "Copy: {}".format(args.origin_path)
+    print "To: {}".format(args.target_server)
+    print "Path: {}".format(args.target_path)
+
+    print "\nRead the above input and confirm typing yes"
```
Comment (iuriinedostup) [inline tools/data/transfer.py]: can you wrap up by quotes the word "yes" just to highlight what user should type?

File: tools/data/transfer.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    target_server.run(create_data_dir)
+
+    os.system(
+        "scp -r {}/. {}:{}/.".format(
+            local_dir, args.target_server, transfer_path
+        )
+    )
+
+    # we have the content, yay, now move it to final dir.
+    copy_content_final_dest = "cd {} && sudo cp -rp . {}".format(
```
Comment (iuriinedostup) [inline tools/data/transfer.py]: maybe better move data rather than copy? what do you think?

Comment (ssola) [inline tools/data/transfer.py]: Yep, it makes sense!

**Comment by iuriinedostup** [comment]
+1

**Pull request merged**